sql/moz-fx-data-shared-prod/subscription_platform_derived/stripe_subscriptions_history_v1/query.sql

```diff
@@ -206,7 +206,9 @@ customers AS (
       pre_fivetran_customers.address_country
     ) AS address_country,
     COALESCE(
-      IF(LENGTH(customers.address_state) = 2, customers.address_state, NULL),
+      IF(LENGTH(customers.address_state) IN (0, 2), customers.address_state, NULL),
```

Comment:
Why allow zero-length strings for state here, which would prevent the coalesce from falling back to the other data sources?

Reply:
I think I completely misread this as a nullif or something which also doesn't make any sense.  I don't remember what I saw when I changed this